plans/dns_jenkinsio.tf

```diff
@@ -43,15 +43,16 @@ locals {
 
   jenkinsio_cname_records = {
     # Azure
-    accounts         = "public.aks.jenkins.io"
-    "archives.azure" = "public.aks.jenkins.io"
-    "mirror.azure"   = "public.aks.jenkins.io"
-    get              = "public.aks.jenkins.io"
-    javadoc          = "public.aks.jenkins.io"
-    plugins          = "dualstack.d.sni.global.fastly.net"
-    reports          = "public.aks.jenkins.io"
-    www              = "dualstack.d.sni.global.fastly.net"
-    uplink           = "public.aks.jenkins.io"
+    accounts              = "public.aks.jenkins.io"
+    "archives.azure"      = "public.aks.jenkins.io"
+    "mirror.azure"        = "public.aks.jenkins.io"
+    get                   = "public.aks.jenkins.io"
+    javadoc               = "public.aks.jenkins.io"
+    plugins               = "dualstack.d.sni.global.fastly.net"
+    reports               = "public.aks.jenkins.io"
+    www                   = "dualstack.d.sni.global.fastly.net"
+    uplink                = "public.aks.jenkins.io"
+    distribution_service  = "public.aks.jenkins.io"
```

Comment:
I really don't think you want a domain called:

distribution_service.jenkins.io

(it's not even valid for ssl certs https://stackoverflow.com/a/54670381/4951015)

please come up with a better name